auth-sample/src/main/res/layout/activity_main.xml

```diff
@@ -20,126 +20,69 @@
     under the License.
 -->
 <android.support.design.widget.CoordinatorLayout
-    android:id="@+id/activity_main"
     xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/activity_main"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context="com.spotify.sdk.android.authentication.sample.MainActivity">
 
-    <android.support.constraint.ConstraintLayout
```

Comment:
What was the change here? Does not seem related to the initial problem at the first sight.